src/interface.jl

```diff
@@ -133,10 +135,13 @@ function save!!(
     iteration::Integer,
     ::AbstractModel,
     ::AbstractSampler,
-    ::Integer;
+    N::Integer;
```

Comment:
We don't need the number of iterations here in the default definition (users with custom implementations might want to use it):
```suggestion
    ::Integer;
```

Reply:
This is needed for `sizehint!`.

Reply:
But we don't want to use `sizehint!` here. We just push.